### PR TITLE
added sidebar

### PR DESCRIPTION
**Pull request by rubikuo**
I added sidebar. is ok to merge

### CODE REVIEW

**Comment by Hirschbaum** [comment]
it is ok

File: src/App.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ function App() {
   return (
     <div className="App">
         <h1>GROUP PROJECT</h1>
+        <p>Hello</p>
```
Comment (rubikuo) [inline src/App.js]: I think this can be better

Comment (Hirschbaum) [inline src/App.js]: <Sidebar />

Comment (rubikuo) [reply src/App.js]: hello any one

Comment (devinardya) [reply src/App.js]: ok!

Comment (devinardya) [reply src/App.js]: ok!

Comment (Hirschbaum) [reply src/App.js]: line 8:
<Sidebar />

**Pull request merged**